CODEOWNERS

```diff
@@ -2,5 +2,8 @@
 # Each line is a file pattern followed by one or more owners,
 # the last matching pattern has the most precendence.
 
+# Alumni maintainers
+# @kjdelisle @loay @ssh24
+
 # Core team members from IBM
-* @kjdelisle @jannyHou @loay @b-admike @ssh24 @virkt25 @dhmlau
+* @jannyHou @b-admike @virkt25 @dhmlau
```

Comment:
i see how it is 😡

Reply:
oh oops. i was just removing people who do not belong in this category and didn't think of who's left.  :)